docs/kb/developer-resources/create-promote-release.md

```diff
@@ -23,6 +23,20 @@ Next, you should have the YML that you want to promote. One common method to gen
 template values indicating the image tags to use. When your build server creates a new Docker image, it’s easy to
 generate a new YAML programmatically.
 
+## Test Your App YAML Changes
+
+When creating new releases during an automated process we recommend testing your app YAML first. This will allow you to validate, and if there any errors during validation exit your build process prior to creating a new release sequence.
+
+To validate your app YAML we use the same call as updating but with a dry_run flag. This will let you look for errors without updating your application. On success the response will be HTTP 200 OK, if there is a problem with your app YAML the service will return a HTTP 400 response with a JSON payload indicating the error. A HTTP 403 response indicates that the sequence your trying to update has already been promoted in which case create a new release and retry the dry run.
```

Comment:
`sequence you're trying to...`
`This will allow you to find any errors without...`
`On success the response will be HTTP 200 OK. If there is...`
`in which case you would create a new release and ...`